index.html

```diff
@@ -2,20 +2,134 @@
+        <link rel="icon" href="Favicon.png" type="image/png">
+        <link rel="stylesheet" href="css/normalize.css">
+        <link rel="stylesheet" href="css/styles.css">
+        <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
+        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
     </head>
     <body>
         <!--[if lt IE 7]>
             <p class="browsehappy">You are using an <strong>outdated</strong> browser. Please <a href="#">upgrade your browser</a> to improve your experience.</p>
         <![endif]-->
+        <header>
+            <div class="top-header sticky">
+                <div class="brand">
+                    <i class="icon inline">
+                        <svg xmlns="http://www.w3.org/2000/svg" width="66" height="10" viewBox="0 0 73.94 14.19">
+                            <polyline points="2.83 2.83 11.36 11.36 19.9 2.83 28.43 11.36 36.97 2.83 45.5 11.36 54.04 2.83 62.58 11.36 71.11 2.83" style="stroke-dashoffset: 2e-05; stroke-dasharray: none;"></polyline>
+                        </svg>
+                    </i>
+                    <p class="name inline">Audrey Hayes</p>
+                </div>
+
+                <nav class="block">
+                    <ul class="hidden-md-down">
+                        <li><a href="https://github.com/snack-attack"><i class="fab fa-github"></i></a></li>
+                        <li><a href="https://codepen.io/snack-attack/"><i class="fab fa-codepen"></i></a></li>
+                        <li><a href="https://dev.to/snackattack"><i class="fab fa-dev"></i></a></li>
+                    </ul>
+                </nav>
+            </div>
+        </header>
         
-        <script src="" async defer></script>
+        <section class="howdy">
+            <h1 class="block">Ho<span class="block">wdy<span class="dot">!</span></span></h1>
+            <p>My name is <span class="highlight">Audrey</span> and I'm a <span class="highlight">fullstack developer</span>.</p>
+        </section>
+        
+        <section class="work">
+            <div class="work-title">
+                <h1 class="work-header"><span class="block">Port</span><span class="block">folio<span class="dot">.</span></span></h1>
+                <p>A selection of client case studies and personal projects.</p>
+            </div>
+            <div>
+                <div class="work-card">
+                    <div class="meta">
+                        <img src="assets/img/workcard-backgrounds/5.jpg" class="photo" alt="photo of colored stucco-like texture">
+                    </div>
+                    <div class="description">
+                        <h2>Site Redesign & PHP Plugin</h2>
```

Comment:
id use a h3 and a h2 here, as a general rule of thumb its good to only have 1 h1 per page